app/models/school.rb

```diff
@@ -140,4 +146,38 @@ def dfe_number
       establishment_number,
     ].join("/")
   end
+
+  def secure_unit?
+    school_type_group == "other" && school_type == "secure_unit"
```

Comment:
This is still checking `school_type_group`, which I think we agreed should be removed.

Reply:
yeah, must be a rebase goof, sorry.